platform/core/src/utils/isDisplaySetReconstructable.js

```diff
@@ -31,37 +33,43 @@ export default function isDisplaySetReconstructable(instances) {
 
 function processMultiframe(instance) {
```

Comment:
instance not being used?

Reply:
no since the method is still not implemented (we do not have any support for multi frames datasets). However I would leave the input to point for future dev that we need to use that instance for implementing the checks.